src/elements/Header.scss

```diff
@@ -1,3 +1,12 @@
+.header {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  max-width: 989px;
```

Comment:
on full screen

Reply:
on full screen - i have this changes in later pull requests